resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -497,6 +500,8 @@ private[spark] class ApplicationMaster(args: ApplicationMasterArguments) extends
         // if the user app did not create a SparkContext.
         throw new IllegalStateException("User did not initialize spark context!")
       }
+      // After initialisation notify user class thread to continue
```

Comment:
nit: rest of the code uses American spelling ("initialization"), so this should be consistent.

Reply:
Fixed and switched to US spell checker.

Reply:
You should remove this comment and add one to the `resumeDriver` method.

Reply:
Moved.